lib/components/map/point-popup.js

```diff
@@ -10,10 +14,20 @@ const PopupTitle = styled.div`
   margin-bottom: 6px;
 `
 
-export default function MapPopup ({
+function MapPopup ({
   mapPopupLocation,
-  onSetLocationFromPopup
+  onSetLocationFromPopup,
+  setMapZoom,
+  zoom
 }) {
+  // Zoom out if zoomed in very far
+  useEffect(() => {
+    if (zoom > 15) {
```

Comment:
Is this not just the zoom from config? (looking at mapstatetoprops)

Reply:
Yes it is, but `setMapZoom` updates this config value which then gets passed into leaflet